doc/dev/release.md

```diff
@@ -12,11 +12,11 @@ Instructions and notes for preparing and publishing a release.
 
     npm install
     npm run build
+    npm run build-css
```

Comment:
The build-css step is one of the steps involved in `npm run build`:
```
    "build": "for target in build-css build-reveal reset-reveal; do npm run $target; done",
```

I may be the one who removed this in the first place, because

Reply:
I copied this from an old branch... thanks for catching it! Fixed in new PR #467.